terraform/aks/databases.tf

```diff
@@ -0,0 +1,15 @@
+module "redis" {
+  source = "git::https://github.com/DFE-Digital/terraform-modules.git//aks/redis?ref=testing"
+
+  namespace             = var.namespace
```

Comment:
do we need to define this in the `variable.aks`?

Reply:
Yeah, that's later in the chain 🤦🏽‍♂️ I'm all for just dropping these PRs in favour of #3587